api/osrd_infra/migrations/__init__.py

```diff
@@ -1,18 +1,33 @@
+import enum
+import itertools
+import re
+from typing import Optional, NamedTuple
+from venv import create
 import django
 from django.db import migrations, models
 
 
-def run_sql_complexe_add_foreign_key(table_name: str, field_name: str, model_name: str, link_model: str, link_table: str, nullable: bool = False):
+def run_sql_complexe_add_foreign_key(
```

Comment:
```suggestion
def run_sql_complex_add_foreign_key(
```

Reply:
Thanks for the review, the diff is from the formatter so changing the name of the function will impact other files that are not related to the search engine